taskcat/_cli_modules/install.py

```diff
@@ -0,0 +1,171 @@
+        generated
+        :param wait: if enabled, taskcat will wait for stack to complete before exiting
+        """
+        boto3_cache = Boto3Cache()
+        if not name:
+            name = generate_name()
+        if region == "default":
+            region = boto3_cache.get_default_region(profile_name=aws_profile)
+        path = Path(package).resolve()
+        if Path(package).resolve().is_dir():
+            package_type = "local"
+        elif "/" in package:
+            package_type = "github"
+        else:  # assuming it's an AWS Quick Start
+            package_type = "github"
+            package = f"aws-quickstart/quickstart-{package}"
+        if package_type == "github":
+            if package.startswith("https://") or package.startswith("git@"):
+                url = package
+                org, repo = (
+                    package.replace(".git", "").replace(":", "/").split("/")[-2:]
+                )
+            else:
+                org, repo = package.split("/")
+                url = f"https://github.com/{org}/{repo}.git"
+            path = Install.PKG_CACHE_PATH / org / repo
+            LOG.info(f"fetching git repo {url}")
+            self._git_clone(url, path)
+            self._recurse_submodules(path, url)
+        config = Config.create(
+            args={"project": {"regions": [region]}},
+            project_config_path=(path / ".taskcat.yml"),
+            project_root=path,
+        )
+        # only use one region
+        for test_name in config.config.tests:
+            config.config.tests[test_name].regions = config.config.project.regions
+        # if there's no test called default, take the 1st in the list
+        if "default" not in config.config.tests:
+            config.config.tests["default"] = config.config.tests[
+                list(config.config.tests.keys())[0]
+            ]
+        # until install offers a way to run different "plans" we only need one test
+        for test_name in list(config.config.tests.keys()):
+            if test_name != "default":
+                del config.config.tests[test_name]
+        buckets = config.get_buckets(boto3_cache)
+        stage_in_s3(buckets, config.config.project.name, path)
+        regions = config.get_regions(boto3_cache)
+        templates = config.get_templates(project_root=path, boto3_cache=boto3_cache)
+        parameters = config.get_rendered_parameters(buckets, regions, templates)
+        tests = config.get_tests(path, templates, regions, buckets, parameters)
+        tags = [Tag({"Key": "taskcat-installer", "Value": name})]
+        stacks = Stacker(config.config.project.name, tests, tags=tags)
+        stacks.create_stacks()
+        LOG.error(
+            f" {stacks.uid.hex}",
+            extra={"nametag": "\x1b[0;30;47m[INSTALL_ID  ]\x1b[0m"},
+        )
+        LOG.error(f" {name}", extra={"nametag": "\x1b[0;30;47m[INSTALL_NAME]\x1b[0m"})
```

Comment:
Doing the escape characters (for custom colors) here is not scalable IMO;

Reply:
yup, total shortcut